codec-http2/src/main/java/io/netty/handler/codec/http2/Http2ConnectionHandler.java

```diff
@@ -153,6 +134,10 @@ public void channelActive(ChannelHandlerContext ctx) throws Exception {
 
     @Override
     public void handlerAdded(ChannelHandlerContext ctx) throws Exception {
+        // Initialize the encoder and decoder.
+        encoder.lifecycleManager(this);
```

Comment:
Ok wasn't sure if we wanted to do the same here...but I guess we want to at least ensure they have a consistent lifecycle manager.